### PR TITLE
Set --max_old_space_size to match memory allocation in launch yml

### PR DESCRIPTION
**Pull request by peggyl**
I trust @rgarcia. 😁

### CODE REVIEW

File: launch/gearman-admin.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ dependencies:
 - gearmand
 resources:
   cpu: 0.1
+  # plesae keep this in sync with --max_old_space_size in package.json
```
Comment (natebrennand) [inline launch/gearman-admin.yml]: s/plesae/please

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
   },
   "scripts": {
     "dev-server": "npm install && node_modules/node-dev/bin/node-dev server.coffee",
+    "start": "node_modules/coffee-script/bin/coffee --max_old_space_size=500 server.coffee",
```
Comment (natebrennand) [inline package.json]: should this be `512`?

**Comment by rgarcia** [comment]
👍

**Comment by peggyl** [comment]
Deploy crashing; reverting until I get a fix out.

File: package.json [hunk 3]
```diff
@@ -29,6 +29,7 @@
   },
   "scripts": {
     "dev-server": "npm install && node_modules/node-dev/bin/node-dev server.coffee",
+    "start": "./node_modules/coffee-script/bin/coffee --max_old_space_size=500 server.coffee",
```
Comment (rgarcia) [inline package.json]: ah, you need `--nodejs` here as well (see https://github.com/Clever/instant-login-api/blob/master/run.sh#L3)

**Pull request merged**